gt/lua_lpm.c

```diff
@@ -366,6 +366,70 @@ l_lpm6_lookup(lua_State *l)
 	return 1;
 }
 
+/*
+ * Takes an array of uint8_t (IPv6 address) and masks it using the depth.
+ * It leaves untouched one bit per unit in the depth variable
+ * and set the rest to 0.
```

Comment:
I don't understand what this sentence is saying, but it seems to be redundant with the previous sentence.